pkg/api/common.go

```diff
@@ -84,10 +85,16 @@ func (a *Auth) Validate() error {
 	return nil
 }
 
+type HighAvailability struct {
```

Comment:
This seems a configuration for `ha` package. Based on what we do for other packages, we have a `config.go` for all modules and we should do the same here. Essentially, All changes here goes in `ha/config.go`. Earlier, it was simply a bool so we didn't have in the `ha` package.

You can take `rules/config.go` for reference.

Reply:
Done!